client/my-sites/controller.js

```diff
@@ -202,12 +202,18 @@ function isPathAllowedForDomainOnlySite( path, slug, primaryDomain ) {
 
 function onSelectedSiteAvailable( context ) {
 	const { getState } = getStore( context );
-	const selectedSite = getSelectedSite( getState() );
+	const state = getState();
```

Comment:
A somewhat irrelevant nit: using `getState` throughout the function is a bit safer, because of the embedded async callbacks. We' be been burned by assigning to the `state` variable before:

https://github.com/Automattic/wp-calypso/pull/17512#discussion_r135450266

Reply:
I can change this back, but this is done within the scope of the callback. Are you saying we have additional async calls in this function?

Reply:
@gwwar It's OK, it just made me remember the last attempt at a similar innocent cleanup.